contrib/demo/workspaceKubectlPlugin-script/workspaceKubectlPlugin

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env bash
+
+# Copyright 2021 The KCP Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+DEMO_DIR="$( dirname "${BASH_SOURCE[0]}" )"
+DEMOS_DIR="$(cd "${DEMO_DIR}/.."; pwd)"
+
+. ${DEMOS_DIR}/demo-magic
+
+ROOT_DIR="$( cd ${DEMOS_DIR}/../.. && pwd)"
```

Comment:
If all of this preamble is shared with others, why not put it into a shared file you `source`?